builtin/providers/google/resource_compute_backend_service.go

```diff
@@ -0,0 +1,410 @@
+		Update: resourceComputeBackendServiceUpdate,
+		Delete: resourceComputeBackendServiceDelete,
+
+		Schema: map[string]*schema.Schema{
+			"backend": &schema.Schema{
+				Type: schema.TypeSet,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"balancing_mode": &schema.Schema{
+							Type:     schema.TypeString,
+							Optional: true,
+							Default:  "UTILIZATION",
+						},
+						"capacity_scaler": &schema.Schema{
+							Type:     schema.TypeFloat,
+							Optional: true,
+							Default:  1,
+						},
+						"description": &schema.Schema{
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"group": &schema.Schema{
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"max_rate": &schema.Schema{
+							Type:     schema.TypeInt,
+							Optional: true,
+						},
+						"max_rate_per_instance": &schema.Schema{
+							Type:     schema.TypeFloat,
+							Optional: true,
+						},
+						"max_utilization": &schema.Schema{
+							Type:     schema.TypeFloat,
+							Optional: true,
+							Default:  0.8,
+						},
+					},
+				},
+				Optional: true,
+				Set:      resourceGoogleComputeBackendServiceBackendHash,
+			},
+
+			"description": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+
+			"health_checks": &schema.Schema{
+				Type:     schema.TypeSet,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+				Required: true,
+				Set:      schema.HashString,
+			},
+
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+				ValidateFunc: func(v interface{}, k string) (ws []string, errors []error) {
+					value := v.(string)
+					re := `^(?:[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?)$`
+					if !regexp.MustCompile(re).MatchString(value) {
+						errors = append(errors, fmt.Errorf(
+							"%q (%q) doesn't match regexp %q", k, value, re))
+					}
+					return
+				},
+			},
+
+			"port_name": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+
+			"protocol": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+
+			"timeout_sec": &schema.Schema{
+				Type:     schema.TypeInt,
+				Optional: true,
+				Computed: true,
+			},
+
+			"fingerprint": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"self_link": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func resourceComputeBackendServiceCreate(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+
+	hc := d.Get("health_checks").(*schema.Set).List()
+	healthChecks := make([]string, 0, len(hc))
+	for _, v := range hc {
+		healthChecks = append(healthChecks, v.(string))
+	}
+
+	service := compute.BackendService{
+		Name:         d.Get("name").(string),
+		Fingerprint:  resource.PrefixedUniqueId("tf-gce-bs-"),
```

Comment:
Agreed, I guess I should just RTFM sometimes :smiley:  https://cloud.google.com/compute/docs/reference/latest/backendServices#resource

> This field will be ignored when inserting a BackendService. An up-to-date fingerprint must be provided in order to update the BackendService.

I'll leave it to be computed.